lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -102,6 +104,17 @@ def _clear_redis_state():
     subprocess.call("service magma@redis stop".split())
 
 
+def _flushall_redis():
```

Comment:
Is there any case where we need to flush all redis state? I think `clear_redis_state` already clears only the relevant stateless services data on redis

Reply:
The use case is to have an option to start fresh. Clean only targets specific keys.

Also, I see `clean` option difficult to maintain: the moment we include a new key we have to remember to add that new key to the clean option in this script. Instead with `flush all` we do know the state is always clean after running it